pkg/armhelpers/converter.go

```diff
@@ -10,11 +10,10 @@ import (
 // DeepCopy dst and src should be the same type in different API version
 // dst should be pointer type
 func DeepCopy(dst, src interface{}) error {
-	defer func() error {
```

Comment:
Where were these deferred `err` responses going prior to this change? Does the caller to `DeepCopy` receive them?

Reply:
I think they were being ignored, because the only way to return a value from a `defer` in a func is a named result parameter. I'll try that change instead, because it would preserve the intent here and hopefully also placate the linter.

Reply:
I changed this and the other `DeepCopy`...um...copy... as specified above.